spring-ai-commons/src/main/java/org/springframework/ai/template/ValidationMode.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023-2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.ai.template;
+
+/**
+ * Validation modes for template renderers.
+ *
+ * @author Thomas Vitale
+ * @since 1.0.0
+ */
+public enum ValidationMode {
```

Comment:
@markpollack  I would keep this internal to StTemplateRenderer since I'm not sure it's generic enough to be useful across template engines, for now. We can re-evaluate once we have concrete examples of different template engines. What do you think?